test/unit/specs/vislib/visualizations/tile_map.js

```diff
@@ -94,6 +98,55 @@ define(function (require) {
           });
         });
 
+        describe('nearestFeature method', function () {
+          it('should return an object', function () {
+            vis.handler.charts.forEach(function (chart) {
+              var lat = (Math.random() * 180) - 90;
+              var lng = (Math.random() * 360) - 180;
+              var point = L.latLng(lat, lng);
+              var mapData = chart.chartData.geoJson;
+              expect(_.isObject(chart.nearestFeature(point, mapData))).to.be(true);
+              expect(chart.nearestFeature(point, mapData).type).to.be('Feature');
+            });
+          });
+        });
+
+        describe('tooltipProximity method', function () {
+          it('should return true', function () {
+            vis.handler.charts.forEach(function (chart) {
+              var zoom = _.random(1, 12);
+              var mapData = chart.chartData.geoJson;
+              var i = _.random(0, mapData.features.length - 1);
+              var feature = mapData.features[i];
+              var point = feature.properties.latLng;
+              var map = chart.maps[0];
+              expect(chart.tooltipProximity(point, zoom, feature, map)).to.be(true);
+            });
+          });
+          it('should return false', function () {
```

Comment:
Can you make the it() descriptions more descriptive?

What should return true and false? Under what conditions?